build.gradle

```diff
@@ -40,9 +40,14 @@ class RenderCurriculumTask extends AsciidoctorTask {
     outputDir = new File("./build/")
     separateOutputDirs = false
     backends 'pdf', 'html5'
+
+    def fileVersion = project.file("./document.version").text + "-" + language
```

Comment:
… and pull this out into a function (if Gradle supports that).

Reply:
Well, I now use `project.version` instead, which is assigned with the same value in line 19.